### ISSUE TITLE
Can't reboot into Recalbox

### ISSUE THREAD

**Issue by nidico100** [matthuisman/kodi-repo#4]
I installed a PINN update and Recalbox Update, right after booting into LibreELEC I installed the bootback for Recalbox, now I can't boot into Recalbox anymore. I can ssh to it but can't see anything. Also bootin into LibreELEC via PINN is luck. Sometimes it works, sometimes not.
EDIT: booting into LibreELEC works no for some reason, but Recalbox still does not show anything.

**Comment by matthuisman** [comment]
I think this maybe due to the latest PINN NOOBS versions.
There have been a bunch of changes that affect booting / rebooting into OS's.

Let me get back to you.

**Comment by matthuisman** [comment]
I have confirmed using 2.4.5f of PINN works for me.

To downgrade without re-installing the OS's:

1) Put SD card into PC
2) Copy recovery.cmdline from SD card to desktop
2) Delete ALL files / folders on SD card EXCEPT recovery.cmdline
3) Download 2.4.5f: https://sourceforge.net/projects/pinn/files/archive/pinn-245f.zip/download
4) Unzip it
5) Copy all it's contents EXCEPT recovery.cmdline to SD card

Basically you want to replace all the files except for recovery.cmdline.

This won't be required once Recalbox does it's next update to make it compatible with the new bootcode found in latest versions of PINN & NOOBS
